client/src/components/AccordionTab.jsx

```diff
@@ -32,7 +32,13 @@ const Tab = (props) => {
       <TabContainer>
         <TabTitle onClick={props.updateTabDisplay}> {props.title}</TabTitle>
         <TabContentContainer >
-          <AccordionTabContent care={props.care} tabTitle={props.title} tabContent={props.content} />
+          <AccordionTabContent
+          fabric={props.fabric}
+          care={props.care}
+          tabTitle={props.title}
+          tabContent={props.content}
+          features={props.features}
```

Comment:
I'm a fan of a little destructuring here to eliminate the reuse of the props object.

Reply:
Yes! I need to make that a habit ASAP 👍

Reply:
Yes, not much to add here, but agree to Garrett!